tools/cli/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+name = "mentat_cli"
+version = "0.0.1"
+
+[lib]
+name = "mentat_cli"
+path = "src/mentat_cli/lib.rs"
+
+[[bin]]
+name = "mentat_cli"
+doc = false
+test = false
+
+[dependencies]
```

Comment:
I just bumped some dependencies in https://github.com/mozilla/mentat/commit/c600152d781f697434389edf20df2948f43426a2 — could you match those?